sqlglot/dialects/spark.py

```diff
@@ -101,6 +101,7 @@ class Generator(Hive.Generator):
             exp.Reduce: rename_func("AGGREGATE"),
             exp.StructKwarg: lambda self, e: f"{self.sql(e, 'this')}: {self.sql(e, 'expression')}",
             HiveMap: _map_sql,
+            exp.PartitionedByProperty: lambda self, e: f"PARTITIONED BY {self.sql(e, 'value')}",
```

Comment:
this is not necessary, spark is a subclass of hive

Reply:
fixed